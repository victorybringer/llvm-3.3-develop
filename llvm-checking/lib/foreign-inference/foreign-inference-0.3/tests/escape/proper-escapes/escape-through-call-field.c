struct S {
  int * sP;
  int * sQ;
};

extern int * g;

void fldEsc(struct S * s) {
  g = s->sP;
}

void f(int * p) {
  struct S s;
  s.sP = p;
  fldEsc(&s);
}
