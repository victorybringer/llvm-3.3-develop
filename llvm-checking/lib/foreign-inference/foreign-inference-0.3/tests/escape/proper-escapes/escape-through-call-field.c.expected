fromList [ ("@fldEsc", fromList [(DirectEscape, "%s.<0>")])
         , ("@f", fromList [(DirectEscape, "%p")])
         ]
